api/v1alpha/README.md

```diff
@@ -675,97 +641,116 @@ Class | Method | HTTP request | Description
+ - [PaginatedAWSPeerVpc](docs/PaginatedAWSPeerVpc.md)
+ - [PaginatedAlert](docs/PaginatedAlert.md)
+ - [PaginatedAlertConfig](docs/PaginatedAlertConfig.md)
+ - [PaginatedApiApiUser](docs/PaginatedApiApiUser.md)
+ - [PaginatedApiAppUser](docs/PaginatedApiAppUser.md)
+ - [PaginatedApiAtlasCheckpoint](docs/PaginatedApiAtlasCheckpoint.md)
+ - [PaginatedApiAtlasDatabaseUser](docs/PaginatedApiAtlasDatabaseUser.md)
+ - [PaginatedApiAtlasDiskBackupExportJob](docs/PaginatedApiAtlasDiskBackupExportJob.md)
+ - [PaginatedApiAtlasProviderRegions](docs/PaginatedApiAtlasProviderRegions.md)
+ - [PaginatedApiAtlasServerlessBackupRestoreJob](docs/PaginatedApiAtlasServerlessBackupRestoreJob.md)
+ - [PaginatedApiAtlasServerlessBackupSnapshot](docs/PaginatedApiAtlasServerlessBackupSnapshot.md)
+ - [PaginatedApiInvoice](docs/PaginatedApiInvoice.md)
+ - [PaginatedApiUserAccessList](docs/PaginatedApiUserAccessList.md)
+ - [PaginatedAppUser](docs/PaginatedAppUser.md)
+ - [PaginatedAtlasGroup](docs/PaginatedAtlasGroup.md)
+ - [PaginatedAzurePeerNetwork](docs/PaginatedAzurePeerNetwork.md)
+ - [PaginatedBackupSnapshot](docs/PaginatedBackupSnapshot.md)
+ - [PaginatedBackupSnapshotExportBucket](docs/PaginatedBackupSnapshotExportBucket.md)
+ - [PaginatedCloudBackupReplicaSet](docs/PaginatedCloudBackupReplicaSet.md)
+ - [PaginatedCloudBackupRestoreJob](docs/PaginatedCloudBackupRestoreJob.md)
+ - [PaginatedCloudBackupShardedClusterSnapshot](docs/PaginatedCloudBackupShardedClusterSnapshot.md)
+ - [PaginatedCloudProviderContainer](docs/PaginatedCloudProviderContainer.md)
+ - [PaginatedClusterDescriptionV15](docs/PaginatedClusterDescriptionV15.md)
+ - [PaginatedDatabase](docs/PaginatedDatabase.md)
+ - [PaginatedDiskPartition](docs/PaginatedDiskPartition.md)
+ - [PaginatedGCPPeerVpc](docs/PaginatedGCPPeerVpc.md)
  - [PaginatedHostViewAtlas](docs/PaginatedHostViewAtlas.md)
- - [PaginatedLegacyClusterView](docs/PaginatedLegacyClusterView.md)
- - [PaginatedNetworkAccessView](docs/PaginatedNetworkAccessView.md)
- - [PaginatedOnlineArchiveView](docs/PaginatedOnlineArchiveView.md)
- - [PaginatedOrgGroupView](docs/PaginatedOrgGroupView.md)
- - [PaginatedOrganizationView](docs/PaginatedOrganizationView.md)
- - [PaginatedPipelineRunView](docs/PaginatedPipelineRunView.md)
- - [PaginatedPrivateLinkConnectionView](docs/PaginatedPrivateLinkConnectionView.md)
- - [PaginatedRestoreJobView](docs/PaginatedRestoreJobView.md)
- - [PaginatedServerlessInstanceDescriptionView](docs/PaginatedServerlessInstanceDescriptionView.md)
- - [PaginatedSnapshotView](docs/PaginatedSnapshotView.md)
- - [PaginatedTeamRoleView](docs/PaginatedTeamRoleView.md)
- - [PaginatedTeamView](docs/PaginatedTeamView.md)
- - [PaginatedTenantRestoreView](docs/PaginatedTenantRestoreView.md)
- - [PaginatedTenantSnapshotView](docs/PaginatedTenantSnapshotView.md)
- - [PaginatedUserCertView](docs/PaginatedUserCertView.md)
+ - [PaginatedLegacyCluster](docs/PaginatedLegacyCluster.md)
```

Comment:
That one seems like leftover:
I think the pattern should be ApiAtlasPaginated so we need to fix those names in OpenAPI instead of transformation